pointcloud/pointcloud_utils.go

```diff
@@ -104,16 +104,16 @@ func ToBasicOctree(cloud PointCloud) (*BasicOctree, error) {
 	if err != nil {
 		return &BasicOctree{}, err
 	}
-	var iterateError error
+	// var iterateError error
```

Comment:
Don't comment this; I corrected my earlier feedback, you are correct to have this here.